atoms/selectedMeasure.ts

```diff
@@ -0,0 +1,10 @@
+import { atom } from 'recoil';
+
+/**
+ * Atom tracking and controlling the measure selected for
+ * bulk submit by the user
+ */
+export const selectedMeasureState = atom({
+  key: 'selectedMeasureState',
+  default: null as string | null
+});
```

Comment:
`atom` is generic :) 

```suggestion
export const selectedMeasureState = atom<string | null>({
  key: 'selectedMeasureState',
  default: null
});
```